### ISSUE TITLE
How to find untested files

### ISSUE THREAD

**Issue by ronkorving** [gotwarlost/istanbul#428]
Hi,

I'm wondering if there is a way to find out which files in a code base weren't tested at all during the code coverage test. It seems I only get output on the files that were require()d directly or indirectly through my tests. But the files that I don't test at all, those should stick out. Instead, I don't see them at all. Shouldn't they appear with 0 coverage in the report?

**Comment by gotwarlost** [comment]
Did you try the `--include-all-sources` option in the `cover` command. I believe that should do what you want. Reopen if this is not the case

**Comment by ronkorving** [comment]
You've got to be kidding me...
You could've saved some people some time by sending that comment a month-and-a-half ago (see nyc reference above). But shame on me for not digging through the CLI help harder.

Anyway, lesson learned. Why don't you and @bcoe join forces and merge the projects?

**Comment by gotwarlost** [comment]
Sorry, man - I must have missed this question somehow.

**Comment by ronkorving** [comment]
No worries, really :) What do you think about merging with nyc? It's absolutely not my call to make (I'm just a bystander here), but would it not make sense?

**Comment by davglass** [comment]
We have been chatting with @bcoe about some things :)
>
>
I have totally dropped the ball on it, but we are working to move Istanbul
to an org & break it into smaller modules with nyc being one of those.
@bcoe would still own it of course, but it would be an official module :)

When I get some free time back I'll get back on it.


-- 
Sent from Gmail Mobile

**Comment by ronkorving** [comment]
Cool, good luck guys :)
It's important work. Everybody uses these tools, and there's nothing better around.

**Comment by bcoe** [comment]
hey @ronkorving, sorry that I didn't catch this for you either :+1:

@davglass @gotwarlost I would happily move nyc into an org (I've also been pretty darn swamped at npm, so haven't pressed too hard).

I think the differentiating factor of nyc vs. istanbul is this:

* nyc tries to as much as possible be a no configuration command, you just point it at tests and run.
  * because it doesn't care whether the runner uses subprocesses, or other exciting spawning approaches, it cuts down on the domain knowledge that the instrumenter needs to have.
* nyc handles subprocesses, this takes enough annoying jumping through hoops that I think we should probably keep it in a separate module (there are a few exciting tricks being used brought to you by the Internet's @isaacs that I think we should keep out of the core Istanbul module).

Long story short, I think of nyc as a next-to-no-configuration coverage bin that pairs nicely with Istanbul.

Thanks for the amazing work on this project \o/ let me know anything I can do to help in moving towards an organization.
